### PR TITLE
Feature/mvp backend api

### PR DESCRIPTION
**Pull request by wisniewski-patryk**
First look

### CODE REVIEW

File: backend/FactoryOps.Domain/Ports/IWorkItemsRepository.cs
Comment (rymarczyk-tomasz) [inline backend/FactoryOps.Domain/Ports/IWorkItemsRepository.cs:1]: dlaczego dodałeś "?" co to daje?

File: backend/FactoryOps.Api/Mocks/WorkItemRepository.cs
Comment (rymarczyk-tomasz) [inline backend/FactoryOps.Api/Mocks/WorkItemRepository.cs:1]: tutaj uzupełniałeś elementy, które należało uzupełnić?? i w sumie co to ma wykonywać? jakaś pamięć elementów(jakich??)

File: backend/FactoryOps.Domain/Ports/IWorkItemsRepository.cs
Comment (wisniewski-patryk) [reply backend/FactoryOps.Domain/Ports/IWorkItemsRepository.cs:1]: This mark typ as 'can be null'. In this case, we return `null` if we don't have the item what are we looking for

[Here, it's explanation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types)

File: backend/FactoryOps.Api/Mocks/WorkItemRepository.cs
Comment (wisniewski-patryk) [reply backend/FactoryOps.Api/Mocks/WorkItemRepository.cs:1]: yeah, it's kind of a database mock. As long as we haven't decided which one we want to use, we can use this simple 'in-memory' database 😉

**Pull request merged**